src/dev/GraphiQL.tsx

```diff
@@ -0,0 +1,218 @@
+import React from "react";
```

Comment:
Can you change this React import to `import * as React from "react":`. I prefer that one :)

Also can u rename the file to lowercase `graphiql.tsx`? Since all the other files are also named lower-case